test/testcafe/spec/Identify_spec.js

```diff
@@ -173,7 +173,7 @@ test.requestHooks(identifyMock)('should show errors if required fields are empty
   await identityPage.clickNextButton();
   await identityPage.waitForErrorBox();
 
-  await t.expect(identityPage.getIdentifierErrorMessage()).eql('This field cannot be left blank');
+  await t.expect(identityPage.getIdentifierErrorMessage()).match(/This field cannot be left blank/);
```

Comment:
We have to use .match(/..../) because any error messages in Odyssey fields will bring in an "Error:" string that is from Odyssey's `ScreenReaderText` component as text content when trying to use a selector